api/v1alpha3/virtualmachine_types.go

```diff
@@ -236,8 +236,44 @@ type VirtualMachineImageRef struct {
 	Name string `json:"name"`
 }
 
+// VirtualMachineCdromSpec describes the desired state of a CD-ROM device.
+type VirtualMachineCdromSpec struct {
+	// +required
+
+	// ImageName is the Kubernetes object name of the VirtualMachineImage or
+	// ClusterVirtualMachineImage resource of type ISO that is used as the
+	// backing for the CD-ROM device.
+	ImageName string `json:"imageName"`
```

Comment:
Should this be simply `image` and it refer to a `type CdromImageRef` where the name is required and kind is optional?